tools/graph-config-jsdoc.json

```diff
@@ -1,1917 +1,1948 @@
 [
-    {
```

Comment:
Please discard these changes; this file is only meant to be updated upon releasing a new version.